src/app/global/components/sidepanel/sidepanel.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { simpleFadeIn, inOutAnimation } from '../../animations/animations';
+import { parentFadeIn, slideInOutAnimation } from '../../animations/animations';
+
+interface NamedItem {
+  name: string;
+}
+
+interface Option {
+  label: string;
+  icon: string;
+  url: string;
+}
+
+@Component({
+  selector: 'unfetter-side-panel',
+  templateUrl: './sidepanel.component.html',
+  styleUrls: ['./sidepanel.component.scss'],
+  animations: [simpleFadeIn, inOutAnimation, parentFadeIn, slideInOutAnimation]
+})
+export class SidepanelComponent implements OnInit {
+
+    @Input() private width = 320;
+
+    @Input() private collapsible = true;
+
+    @Input() public minimized = false;
+
+    @Input() public headerBackground = '';
+
+    @Input() private item: NamedItem;
```

Comment:
npm run build:prod runs Angular AOT compilation.  Usually AOT likes to bind to public members from template to components, this private instance might cause a problem

Reply:
There's two private members in there, I'll set both to public.